crypto/bio/bss_file.c

```diff
@@ -89,6 +89,10 @@
 
 # if !defined(OPENSSL_NO_STDIO)
 
+#  if defined(OPENSSL_SYS_MSDOS) && !defined(OPENSSL_SYS_WINDOWS)
+#   include <libc/unconst.h>
+static void dosify_filename(const char *filename);
+#  endif
```

Comment:
Hmmm...  come to think of it, considering `file_fopen` is already an `#if` sallad, I'm not sure if suddenly using a radically different solution for DJGPP makes sense right now.  In that case, what's currently in `file_fopen` should be refactored in a similar manner...  and *that's* a bigger job, which I'd make post 1.1.0.

However, the DJGPP solution in this PR isn't quite right, as you remarked...  Working on it.